src/components/ModalPortal.js

```diff
@@ -18,6 +18,7 @@ const ESC_KEY = 27;
 
 export default class ModalPortal extends Component {
   static defaultProps = {
+    shouldCloseOnEsc: true,
```

Comment:
You can move this prop to `Modal.js` and modal will pass to the portal.

Reply:
Well spotted!
I've moved the default prop value :)